mixins/create-edit-view.js

```diff
@@ -173,7 +173,6 @@ export default {
 
         // Remove the labels map if it's empty
         if ( this.value?.metadata?.labels && Object.keys(this.value.metadata.labels || {}).length === 0 ) {
-          this.$set(this.value.metadata, 'annotations', {});
```

Comment:
This is there because I don't think(?) deleting the parent object triggers reactivity on the child keys, but should have said `labels` instead of `annotations`...

Reply:
I guess the labels one doesn't have it anymore either so I'll assume it's fine for now.